src/elements/DataTable.vue

```diff
@@ -126,6 +149,29 @@ export default {
         return col.name
       }
     },
+    sortTable(value) {
+      if (!value.ascending) {
+        this.rows.sort(function(a, b) {
+          var textA = a[value.name.toLowerCase()].toString().toLowerCase()
+          var textB = b[value.name.toLowerCase()].toString().toLowerCase()
```

Comment:
Does this work for 35 and 5?  Aka 5 is less than 35 numerically, but is likely to be greater than 35 in a string comp.

Reply:
Again, nice catch. This only does alphanumeric. We will need to test the type for this, but the previous code I had will work for numeric.